src/main/java/nextstep/ladder/view/ResultView.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.ladder.view;
+
+import nextstep.ladder.domain.LadderBoard;
+import nextstep.ladder.domain.Line;
+import nextstep.ladder.domain.UserNames;
+
+import java.util.List;
+
+public class ResultView {
+    private final static String GAP = "     ";
+    private final static String LINE = "-----";
+    private final static String LADDER = "|";
+    private final static String LADDER_FORMAT = "%6s";
+
+
+    public static void printNames(UserNames userNames) {
+        userNames.getNames().stream().map(name -> String.format(LADDER_FORMAT, name)).forEach(System.out::print);
+        System.out.println();
+    }
+
+    public static void printLadderBoard(LadderBoard ladderBoard) {
+        ladderBoard.getLadderBoard().forEach(line -> {
+            printGap();
+            printRow(line);
+            printLadder();
+            System.out.println();
+        });
+    }
+
+    private static void printRow(Line line) {
+        line.pointToBoolean().forEach(point -> {
+            printLadder();
+            if (point) {
+                printLine();
+            } else {
+                printGap();
+            }
```

Comment:
```suggestion
            if (point) {
                printLine();
            } else {
                printGap();
            }
```
for-if 2뎁스군요.
로직의 행동을 표현해보아요.